docs/src/faqs.md

```diff
@@ -28,33 +26,35 @@ observed(osys)
 Let's solve the system and see how to index the solution using our symbolic
 variables
 ```@example faq1
+u0 = [rn.A => 1.0, rn.B => 2.0, rn.C => 0.0]
+ps = [rn.k₊ => 1.0, rn.k₋ => 1.0]
 oprob = ODEProblem(osys, [], (0.0, 10.0), [])
 sol = solve(oprob, Tsit5())
 ```
 Suppose we want to plot just species `C`, without having to know its integer
 index in the unknown vector. We can do this using the symbolic variable `C`, which
 we can get at in several ways
-```@example faq1
+```julia
 sol[osys.C]
 ```
 or
-```@example faq1
+```julia
 @unpack C = osys
 sol[C]
 ```
 To evaluate `C` at specific times and plot it we can just do
-```@example faq1
+```julia
 t = range(0.0, 10.0, length=101)
 plot(t, sol(t, idxs = C), label = "C(t)", xlabel = "t")
 ```
 If we want to get multiple variables we can just do
-```@example faq1
+```julia
 @unpack A, B = osys
 sol(t, idxs = [A, B])
 ```
 Plotting multiple variables using the SciML plot recipe can be achieved
 like
-```@example faq1
+```julia
 plot(sol; idxs = [A, B])
 ```
```

Comment:
Why are these no longer dynamic?

Reply:
I was trying to reduce runtime, sorry it got confused.

If you are happy with the change above I can reenable these, but if not we will have to figure something else out.

Reply:
Yeah, if you reenable these I am fine to merge. I'd prefer we keep everything dynamic if we can.